File: eRecipes/UI/erecipes_mobile/lib/models/to_do4924.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:erecipes_mobile/models/korisnik.dart';

part 'to_do4924.g.dart';

@JsonSerializable()
class ToDo4924 {
  int? ToDo4924Id;
  String? naziv;
  String? opis;
  DateTime? datumIzvrsenja;
  String? status;
  int? korisnikId;
  Korisnik? korisnik;

  ToDo4924(
    this.korisnikId,
    this.naziv,
    this.opis,
    this.datumIzvrsenja,
    this.status
  );

  factory ToDo4924.FromJson(Map<String, dynamic> json) =>
      _$ToDo4924FromJson(json);

  Map<String, dynamic> toJson() => _$ToDo4924ToJson(this);
}

File: eRecipes/eRecipes.Model/ToDo4924.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRecipes.Model
{
    public class ToDo4924
    {
        public int ToDo4924Id { get; set; }
        public int KorisnikId { get; set; }
        public virtual Korisnik Korisnik { get; set; }
        public string Naziv { get; set; }
        public string Opis { get; set; }
        public DateTime DatumIzvrsenja { get; set; }
        public string Status { get; set; }
    }
}
